admin.php

```diff
@@ -38,131 +43,220 @@ function sqlQuery($statement, $link)
 <html>
 <head>
 <title>OpenEMR Site Administration</title>
-<link rel='STYLESHEET' href='interface/themes/style_sky_blue.css'>
+<link rel="stylesheet" href="public/assets/bootstrap/dist/css/bootstrap.min.css" type="text/css">
+<link rel="stylesheet" href="public/assets/jquery-ui/jquery-ui.css" type="text/css">
+<script type="text/javascript" src="public/assets/jquery/dist/jquery.min.js"></script>
+<script type="text/javascript" src="public/assets/bootstrap/dist/js/bootstrap.min.js"></script>
+<script type="text/javascript" src="public/assets/jquery-ui/jquery-ui.js"></script>
+<link rel="stylesheet" href="public/assets/font-awesome/css/font-awesome.min.css" type="text/css">
+<link rel="shortcut icon" href="public/images/favicon.ico" />
 <style>
-tr.head   { font-size:10pt; background-color:#cccccc; text-align:center; font-weight:bold; }
-tr.detail { font-size:10pt; }
+/*tr.head   { font-size:10pt; background-color:#cccccc; text-align:center; font-weight:bold; }
+tr.detail { font-size:10pt; }*/
 a, a:visited, a:hover { color:#0000cc; text-decoration:none; }
+
+.oe-pull-away{
+        float:right;
+    }
+    .oe-help-x {
+        color: grey;
+        padding: 0 5px;
+    }
+    .oe-superscript {
+        position: relative;
+        top: -.5em;
+        font-size: 70%!important;
+    }
+    .oe-setup-legend{
+        background-color:  WHITESMOKE;
+        padding:0 10px;
+    }
+    button {
+    font-weight:bold;
+    }
+    .button-wait {
+        color: grey;
+        cursor: not-allowed;
+        opacity: 0.6;
+    }
+    @media only screen {
+        fieldset > [class*="col-"] {
+            width: 100%;
+            text-align:left!Important;
+        }
+    }
 </style>
 </head>
 <body>
-<center>
-<p><span class='title'>OpenEMR Site Administration</span></p>
-<table width='100%' cellpadding='1' cellspacing='2'>
- <tr class='head'>
-  <td>Site ID</td>
-  <td>DB Name</td>
-  <td>Site Name</td>
-  <td>Version</td>
-  <td>Action</td>
- </tr>
-<?php
-$dh = opendir($OE_SITES_BASE);
-if (!$dh) {
-    die("Cannot read directory '$OE_SITES_BASE'.");
-}
+    <div class='container'>
+        <div class="row">
+            <div class="col-sm-12">
+                <div class="page-header">
+                    <h2>OpenEMR Multi Site Administration <a class="oe-pull-away oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#676666" title="Click to view Help"><i class="fa fa-question-circle" aria-hidden="true"></i></a></h2>
+                </div>
+            </div>
+        </div>
+        
+        <div class="row">
+            <div class="col-sm-12">
+                <table class='table table-striped'>
+                 <tr class='head'>
+                  <td>Site ID</td>
+                  <td>DB Name</td>
+                  <td>Site Name</td>
+                  <td>Version</td>
+                  <td align='center'>Log In</td>
```

Comment:
Note there are more possible actions here than just Log In:
![image](https://user-images.githubusercontent.com/278968/51590492-f5dbce80-1e9e-11e9-9736-1ab0d89b1092.png)
![image](https://user-images.githubusercontent.com/278968/51590550-2ae82100-1e9f-11e9-9cb9-c355aeedc7c2.png)
![image](https://user-images.githubusercontent.com/278968/51590624-7e5a6f00-1e9f-11e9-9c97-6ee282898f07.png)

Reply:
It would be nice to place these elements in their own column and keep the Log In column separate. Recommend a column to left of Log In called something like Maintenance where these other elements go. And if nothing, then can place something like Up To Date.

Reply:
will revert

Reply:
will work on it

Reply:
quick question 

```
if ($v_database != $database_version) {
    echo "  <td><a href='sql_upgrade.php?site=$sfname'>Upgrade Database</a></td>\n";
} elseif (($v_acl > $database_acl)) {
    echo "  <td><a href='acl_upgrade.php?site=$sfname'>Upgrade Access Controls</a></td>\n";
} elseif (($v_realpatch != $database_patch)) {
    echo "  <td><a href='sql_patch.php?site=$sfname'>Patch Database</a></td>\n";
} else {
    echo "  <td align='center'><a href='interface/login/login.php?site=$sfname' title =' Login to site $sfname' ><i class='fa fa-sign-in fa-lg' aria-hidden='true'></i></a></td>\n";
}
```

Current logic only displays login `if ($v_database != $database_version)`  is not true. Do you want it displayed at all times

Reply:
I could work on displaying a disabled login icon for the not true scenario

Reply:
I like the idea of a disabled login icon the the not true scenario.